src/api/flaskr/api/llm/__init__.py

```
@@ -1,241 +1,305 @@
-        ernie_enabled = False
-        ERNIE_MODELS = []
-else:
-    current_app.logger.warning("ERNIE_API_ID and ERNIE_API_SECRET not configured")
+        return litellm.completion(
+            model=model,
+            messages=messages,
+            stream=True,
+            **params,
+            **kwargs,
+        )
+    except Exception as exc:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)